src/request/components/request-detail-panel-logging.tsx

```diff
@@ -1,49 +1,67 @@
+                                })
+                        }
+                        </div>
+                    </div>
+                    <br/>
                     <table className='table table-bordered table-striped tab-content-item'>
                         <thead>
                             <tr className='table-col-title-group'>
-                                <th width='5%'><span className='table-col-title'>#</span></th>
+                                <th width='5%'><span className='table-col-title'>Ordinal</span></th>
                                 <th width='10%'><span className='table-col-title'>Level</span></th>
-                                <th><span className='table-col-title'>Message</span></th>
+                                <th width='40%'><span className='table-col-title'>Message</span></th>
                                 <th width='10%'><span className='table-col-title'>From Start</span></th>
-                                <th width='10%'><span className='table-col-title'>Duration</span></th>
+                                <th><span className='table-col-title'>Duration</span></th>
                             </tr>
                         </thead>
-                        {this.props.viewModel.messages.map(function(message, index) {
-                            const className = Logging.getRowClass(message);
-
-                            return (
-                                <tr className={className}>
-                                    <td>{index + 1}</td>
-                                    <td>{message.level}</td>
-                                    <td>{message.message}</td>
-                                    <td>-</td>
-                                    <td>-</td>
-                                </tr>);
-                        }) }
                         <tfoot>
                             <tr className='table-body-padding table-col-title-group'><th colSpan='5'></th></tr>
                         </tfoot>
+                        <tbody>
+                        {
+                            messages.map(message => {
+                                return (
+                                    <tr className='tab-logs-data-default' key={message.id}>
+                                        <td>{message.ordinal}</td>
+                                        <td className={Logging.getRowClass(message)}>{message.level}</td>
+                                        <td>{message.message}</td>
+                                        <td>-</td>
+                                        <td>-</td>
+                                    </tr>);
+                            })
+                        }
+                        </tbody>
```

Comment:
I *recall* that it had something to do with the keys attached to each row (needed by React to properly add/remove items).  I can move the `tbody` around and see if it was just a fluke of something else I was doing at the time.